### ISSUE TITLE
Dark mode

### ISSUE THREAD

**Issue by ronreiter** [ronreiter/interactive-tutorials#550]
We need one

**Comment by PBahner** [comment]
Yes you are definitely right. So where to start? Maybe starting with a toggle button to switch between light and dark? Which .css file is for the tutorial pages?

**Comment by ronreiter** [comment]
https://css-tricks.com/dark-modes-with-css/
